test/controllers/trusts.involved.controller.spec.ts

```diff
@@ -0,0 +1,141 @@
+jest.mock('../../src/middleware/authentication.middleware');
+jest.mock('../../src/middleware/navigation/has.trust.middleware');
+jest.mock('../../src/middleware/is.feature.enabled.middleware');
+jest.mock('../../src/middleware/is.feature.enabled.middleware', () => ({
+  isFeatureEnabled: () => (_, __, next: NextFunction) => next(),
+}));
+
+import { constants } from 'http2';
+import { beforeEach, describe, expect, jest, test } from '@jest/globals';
+import { NextFunction, Request, Response } from "express";
+import { Params } from 'express-serve-static-core';
+import { Session } from '@companieshouse/node-session-handler';
+import request from "supertest";
+import app from "../../src/app";
+import { get, post, TRUST_INVOLVED_TEXTS } from "../../src/controllers/trust.involved.controller";
+import { ANY_MESSAGE_ERROR, PAGE_TITLE_ERROR } from '../__mocks__/text.mock';
+import { authentication } from '../../src/middleware/authentication.middleware';
+import { hasTrust } from '../../src/middleware/navigation/has.trust.middleware';
+import { TRUST_INVOLVED_URL } from '../../src/config';
+
+
+describe('Trust Involved controller', () => {
+
```

Comment:
remove line, no line after `{` or before `}` here and below

Reply:
What do you mean here? Line 21 and 23 should be removed?

Reply:
lines 51 65 82 84  156 158